### PR TITLE
issue-864

### PR DESCRIPTION
**Pull request by BalloonWen**
- now logging the status ans stack trace separately
- now will only log the stack trace when TRACE is enabled.

### CODE REVIEW

File: handler/src/main/java/com/networknt/handler/LightHttpHandler.java [hunk 1]
```diff
@@ -43,24 +44,24 @@
...
- 	boolean auditOnError = config != null ? config.getAuditOnError() : false;
- 	boolean auditStackTrace = config != null ? config.getAuditStackTrace() : false;
-    
     /**
      * This method is used to construct a standard error status in JSON format from an error code.
      *
      * @param exchange HttpServerExchange
-     * @param code error code
-     * @param args arguments for error description
+     * @param code     error code
```
Comment (jaswalkiranavtar) [inline handler/src/main/java/com/networknt/handler/LightHttpHandler.java:59]: no need for tab here as per javadoc standard

File: handler/src/main/java/com/networknt/handler/LightHttpHandler.java [hunk 2]
```diff
@@ -43,24 +44,24 @@
...
+     * @param args     arguments for error description
```
Comment (jaswalkiranavtar) [inline handler/src/main/java/com/networknt/handler/LightHttpHandler.java:60]: no need for tab here as per javadoc standard

File: handler/src/main/java/com/networknt/handler/LightHttpHandler.java [hunk 3]
```diff
@@ -72,27 +73,35 @@ default void setExchangeStatus(HttpServerExchange exchange, String code, final O
      * want to bubble up to the caller and eventually to the original caller.
      *
      * @param exchange HttpServerExchange
-     * @param status error status
+     * @param status   error status
```
Comment (jaswalkiranavtar) [inline handler/src/main/java/com/networknt/handler/LightHttpHandler.java:76]: no need for tab here as per javadoc standard

File: handler/src/main/java/com/networknt/handler/LightHttpHandler.java (hunk 1 above)
Comment (BalloonWen) [reply handler/src/main/java/com/networknt/handler/LightHttpHandler.java:59]: This is for align with the HttpServerExchange
Based on the example on Oracle doc https://www.oracle.com/ca-en/technical-resources/articles/java/javadoc-tool.html#format
You can see the "url" "name " in this example is also aligned.
I guess either aligned or not aligned is fine, what do you think?

Comment (jaswalkiranavtar) [reply handler/src/main/java/com/networknt/handler/LightHttpHandler.java:59]: Ok I am fine if according to official documentation, it is acceptable. But i have not seen it much in practice.
Looks good to me.
Apprving

**Review by jaswalkiranavtar — APPROVED** [review]

**Review by stevehu — APPROVED** [review]

**Pull request merged**